setup.py

```diff
@@ -6,6 +6,7 @@
                            'README.rst')
 
 dependencies = [
+    'django-filer',
```

Comment:
I prefer to leave it this way since it's an open source project and it can be used with any version of filer(pbs or non pbs).